### PR TITLE
Directly fly to tasks when dead

### LINKED ISSUE #49: Make neuro autocomplete all remaining tasks when she is dead
When you are a ghost we don't really care about using the neural network anymore, instead we should just go straight to and finish all the remaining tasks.

### PR DESCRIPTION
**Pull request by Taflaxx**
Fly directly to the closest task when the player dies. In case all tasks are completed it will follow a random alive player.

closes #49

### CODE REVIEW

File: Neuro/Movement/MovementHandler.cs [hunk 1]
```diff
@@ -86,6 +91,57 @@ public void GetForcedMoveDirection(ref Vector2 direction)
...
+                    alivePlayers.Add(player);
+
+                } 
+            }
+            System.Random random = new();
+            int i = random.Next(alivePlayers.Count);
+            followPlayer = alivePlayers[i];
+            Info("Now following " + followPlayer.name);
+        }
+        moveToPosition(followPlayer.GetTruePosition());
```
Comment (Alexejhero) [inline Neuro/Movement/MovementHandler.cs:134]: Instead of doing this, use the GhostRole's haunt ability to haunt an impostor

**Review by Alexejhero — DISMISSED** [review]

File: Neuro/Movement/MovementHandler.cs (hunk 1 above)
Comment (Taflaxx) [reply Neuro/Movement/MovementHandler.cs]: CrewmateGhosts now automatically haunt impostors. How should ImpostorGhosts and GuardianAngel be handled? You can't sabotage while haunting and the GuardianAngel doesn't even have the ability to haunt.

**Comment by Taflaxx** [comment]
Impostor and Crewmate ghosts now use the haunt ability to follow an impostor. Since GuardianAngel cant use haunt it will manually follow a random player and use protect on cooldown.

**Review by Alexejhero — COMMENTED** [review]
The code looks good to me, we should test before merging though.

**Comment by Taflaxx** [comment]
I fixed a small error. I tested it again (with dummies) and everything seems to work now. But it would be nice if someone else could also test it (preferably in a multiplayer lobby).

**Review by Alexejhero — APPROVED** [review]

**Comment by Alexejhero** [comment]
Tested and everything seems to work fine.

**Pull request merged**